tests/tests/test_update_control.py

```diff
@@ -149,7 +155,7 @@ def test_update_control_with_broken_map(
         devauth = DeviceAuthV2(auth)
 
         enterprise_no_client.new_tenant_client("control-map-test-container", ttoken)
-        device = MenderDevice(enterprise_no_client.get_mender_clients()[0])
+        MenderDevice(enterprise_no_client.get_mender_clients()[0])
```

Comment:
Either remove this line or create the object but add it to the environment object (so that the logs can be printed in case of errors):

`enterprise_no_client.device = MenderDevice(enterprise_no_client.get_mender_clients()[0])`